# A representation of Gyroscope bias in body frame (x,y,z)

Header header
geometry_msgs/Vector3 bias
